### PR TITLE
Fix feat_proj params

### PR DESCRIPTION
**Pull request by icolwell-as**
Transferred from GitLab. See original PR for details:

https://gitlab.com/autowarefoundation/autoware.ai/core_perception/-/merge_requests/79

### CODE REVIEW

File: trafficlight_recognizer/launch/feat_proj.launch [hunk 1]
```diff
@@ -13,5 +13,4 @@
     <param name="roi_search_min_distance" value="$(arg roi_search_min_distance)"/>
     <param name="roi_search_max_distance" value="$(arg roi_search_max_distance)"/>
   </node>
-
```
Comment (LiyouZhou) [inline trafficlight_recognizer/launch/feat_proj.launch:16]: This line change is no necessary, would prefer if this change is not in this PR.

**Review by LiyouZhou — CHANGES_REQUESTED** [review]
I can see what you did here. but for the sake of clear history and documentation, would you mind adding some details in the commit message about why this change is made.

see https://chris.beams.io/posts/git-commit/#why-not-how for some tips on commit messages.

**Comment by LiyouZhou** [comment]
@icolwell-as looks like all your PRs only contain a link to another PR which also does not contain too much info either. Would you mind just populating the commit messages and PR bodys with a bit more detail. This would help me greatly in review. Thank you very much.

**Comment by icolwell-as** [comment]
@LiyouZhou, my apologies for the sometimes sparse PR explanations. Some of these PRs are left over from when Autoware.AI was transferred from gitlab to github hence the link to other PRs. The rest of the PRs are transferred from our gitlab fork and include links back to the original MR which contains descriptions of the changes and links to issues that the MR is resolving, there are also code review discussions which may help you understand some of the code.

Re-writing the commit history of all these commits will be very time consuming, but improving the PR descriptions is more feasible if that is acceptable to you. Are there some specific PRs that are the most confusing that you would like me to start with?

**Comment by icolwell-as** [comment]
@JWhitleyWork, reminder to re-open (assuming you closed to trigger a re-build).

**Comment by JWhitleyWork** [comment]
@icolwell-as Sorry about that. It got automatically closed due to the issue number being mentioned in a commit message. This PR looks perfectly fine to me. I'll merge once CI passes.

**Review by JWhitleyWork — APPROVED** [review]

**Pull request merged**